src/app/EventManagement.cpp

```diff
@@ -312,9 +315,32 @@ CHIP_ERROR EventManagement::ConstructEvent(EventLoadOutContext * apContext, Even
     VerifyOrExit(apOptions != nullptr, err = CHIP_ERROR_INVALID_ARGUMENT);
     VerifyOrExit(apOptions->mTimestamp.mType != Timestamp::Type::kInvalid, err = CHIP_ERROR_INVALID_ARGUMENT);
 
-    eventDataElementBuilder.Init(&(apContext->mWriter));
-    eventPathBuilder = eventDataElementBuilder.CreateEventPath();
-    err              = eventPathBuilder.GetError();
+    eventReportBuilder.Init(&(apContext->mWriter));
+    // TODO: Update IsUrgent
+    // TODO: Update statusIB
```

Comment:
Could you generate github issues for these and track with `TODO(#xxxx)`?

Reply:
Done, thanks, updated
https://github.com/project-chip/connectedhomeip/issues/11386
https://github.com/project-chip/connectedhomeip/issues/11388